templates/test/ci/cluster-template-prow-intree-cloud-provider.yaml

```diff
@@ -482,6 +483,55 @@ spec:
     windows:
       useHostProcessContainers: {{ hasKey .Cluster.metadata.labels "cni-windows" }}
 ---
+apiVersion: addons.cluster.x-k8s.io/v1alpha1
```

Comment:
Does the in-tree config supersede this? Or should we not be including the CAAPH config at all in this template?

Reply:
Hmm good point, I think it does supersede but it's still not very logical to include it. Since the in tree cloud-providers are deprecated at this point and we're no longer testing them anywhere outside of CAPZ, it might be time to remove this altogether.

Reply:
Removed all in-tree templates